apps/site/pages/en/learn/typescript/run-natively.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Running TypeScript Natively
+layout: learn
+authors: AugustinMauroy
+---
+
+> **⚠️WARNING⚠️:** All content in this article uses Node.js experimental features. Please make sure you are using a version of Node.js that supports the features mentioned in this article. And remember that experimental features can change on future versions of Node.js.
+
+# Running TypeScript Natively
+
+In the previous articles, we learned how to run TypeScript code using transpilation and with a runner. In this article, we will learn how to run TypeScript code using Node.js itself.
+
+## Running TypeScript code with Node.js
+
+Since V22.6.0, Node.js has experimental support for some TypeScript syntax. You can write code that's valid TypeScript directly in Node.js without the need to transpile it first.
+
+So how do you run typed JavaScript code with Node.js?
+
+```bash
+node --experimental-strip-types example.ts
+```
+
+The `--experimental-strip-types` flag tells Node.js to strip the type annotations from the TypeScript code before running it.
+
+And that's it! You can now run typed JavaScript code directly in Node.js without the need to transpile it first, and use TypeScript to catch type-related errors.
+In the future we all hope that this feature will be stable and available in the LTS version of Node.js, so that we can all enjoy it without any additional steps.
+
+## Limitations
+
+At the time of writing, the experimental support for TypeScript in Node.js has some limitations. To allow typescript to run in node.js, our collaborators have chosen to only strip types from the code.
```

Comment:
I think what you are implying here is  that Node.js does not do any type-checking, right?

Reply:
yes, and that it doesn't transform anything